2025/XenSummit/Implementing_Secure_Boot_In_Xen.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: cover
+background: /intro.png
+class: text-center
+routeAlias: implementing-uefi
+
+---
+
+## Implementing UEFI Secure Boot in Xen and Qubes OS
```

Comment:
WRT conference place and title - I assume it's this one: https://cfp.vates.tech/xen-meetup-2025/talk/8JBQKC/